src/pgduckdb_ddl.cpp

```diff
@@ -51,7 +51,7 @@ DuckdbTruncateTable(Oid relation_oid) {
 }
 
 void
-DuckdbHandleDDL(Node *parsetree, const char *queryString) {
+DuckdbHandleDDL(Node *parsetree, const char *) {
```

Comment:
I think here we can just remove this argument, since we control the callsite.

Reply:
Great catch - and that's a good example of why I think this warning is useful :-)